components/brave_new_tab_ui/components/default/editCards/style.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'brave-ui/theme'
+import palette from 'brave-ui/theme/colors'
+
+export const StyledWrapper = styled<{}, 'button'>('button')`
+  margin: 0;
+  width: 100%;
+  border: none;
+  display: block;
+  margin-top: 5px;
+  appearance: none;
+  border-radius: 8px;
+  cursor: pointer;
+  backdrop-filter: blur(23px);
+  padding: 12px 20px 8px 20px;
+  background: linear-gradient(90deg, rgba(33, 37, 41, 0.6) 0%, rgba(33, 37, 41, 0.24) 100%);
+
+  &:hover {
+    background: rgba(33, 37, 41, 0.5);
+  }
+
+  &:focus {
```

Comment:
Can you make this `:focus-visible` so that it only shows for non-mouse users please? Also design seems to show a rounded focus ring, that can be achieved with:

```css
outline: none;

:focus-visible {
  box-shadow: 0 0 0 1px ${p => p.theme.color.brandBrave};
}
```

Reply:
fixed